Source/AsyncGenerator.Tests/ExternalProjects/NHibernate/Fixture.cs

```diff
@@ -150,6 +150,15 @@ private MethodConversion GetMethodConversion(IMethodSymbol symbol)
 					}
 					break;
 
+				case "PositionalParameterSpecification":
+				case "NamedParameterSpecification":
```

Comment:
@maca88 for some reason these 2 methods are not generated for my branch https://github.com/hazzik/nhibernate-core/tree/NH-2088